### ISSUE TITLE
GW Objects not being added to nameserver database

### ISSUE THREAD

**Issue by genghisken** [genghisken/ps1#40]
Originally reported by: **Ken Smith (Bitbucket: [genghisken](https://bitbucket.org/genghisken), GitHub: [genghisken](https://github.com/genghisken))**

----------------------------------------

Although GW objects are getting PS1 names, the QUB aliases are not being added to the nameserver database.

----------------------------------------
- Bitbucket: https://bitbucket.org/genghisken/ps1/issue/40

**Comment by genghisken** [comment]
*Original comment by* **Ken Smith (Bitbucket: [genghisken](https://bitbucket.org/genghisken), GitHub: [genghisken](https://github.com/genghisken))**:

----------------------------------------

The GW database was not added to the database whitelist at the top of addqub_ps1_transient.py.  I fixed the other objects by effectively re-assigning their names.  All the GW aliases are now in the nameserver database.

This problem highlights two major issues.

1.  No-one currently maintains the nameserver database.  This should be migrated to QUB along with any associated HTML generated.
2. The code to grab the names is sitting in a "dead" SVN repository.  This code should be added to this PS1 repo.

I have generated two new issues to cover this work.
